src/calc_rsoil.m

```diff
@@ -1,12 +1,32 @@
-function [PSIs, rsss, rrr, rxx] = calc_rsoil(Rl, DeltZ, Ks, Theta_s, Theta_r, Theta_LL, bbx, m, n, Alpha)
+function [PSIs, rsss, rrr, rxx] = calc_rsoil(Rl, ModelSettings, SoilVariables, VanGenuchten, bbx, GroundwaterSettings)
     % load Constants
     Constants = io.define_constants();
 
+    DeltZ = ModelSettings.DeltZ;
+    Ks = SoilVariables.Ks;
+    Theta_s = VanGenuchten.Theta_s;
+    Theta_r = VanGenuchten.Theta_r;
+    Theta_LL = SoilVariables.Theta_LL;
+    m = VanGenuchten.m;
+    n = VanGenuchten.n;
+    Alpha = VanGenuchten.Alpha;
```

Comment:
Please avoid defining variables two times and use the variable as `ModelSettings.DeltZ` in the equation because they are only used a few times.

Reply:
> Please avoid defining variables two times and use the variable as `ModelSettings.DeltZ` in the equation because they are only used a few times.
> 
Done